Description: Don't make the std:: symbols in webkit public.
Solution based on the work of Mike Hommey in the gtk webkit package.
Author: Sune Vuorela <debian@pusling.com>

---
 src/3rdparty/webkit/Source/WebKit.pri     |    2 ++
 src/3rdparty/webkit/Source/symbols.filter |    5 +++++
 2 files changed, 7 insertions(+)

--- a/src/3rdparty/webkit/Source/WebKit.pri
+++ b/src/3rdparty/webkit/Source/WebKit.pri
@@ -100,6 +100,8 @@ win32-g++* {
 CONFIG -= warn_on
 *-g++*:QMAKE_CXXFLAGS += -Wall -Wextra -Wreturn-type -fno-strict-aliasing -Wcast-align -Wchar-subscripts -Wformat-security -Wreturn-type -Wno-unused-parameter -Wno-sign-compare -Wno-switch -Wno-switch-enum -Wundef -Wmissing-noreturn -Winit-self
 
+*-g++*:QMAKE_LFLAGS += -Wl,--version-script,$$PWD/symbols.filter
+
 # Treat warnings as errors on x86/Linux/GCC
 linux-g++* {
     !CONFIG(standalone_package):isEqual(QT_ARCH,x86_64)|isEqual(QT_ARCH,i386): QMAKE_CXXFLAGS += -Werror
--- /dev/null
+++ b/src/3rdparty/webkit/Source/symbols.filter
@@ -0,0 +1,5 @@
+{
+local:
+_ZSt*;
+_ZNSt*;
+};
